### PR TITLE
refactor: add css tokens

### PR DESCRIPTION
**Pull request by lisalupi**
## Type
- Feature

#### What is expected?

Ultraviolet theme now provides css tokens.
The CSS files are located in "@ultraviolet/themes/public/style".  To import a theme, dark for instance, do : `import '@ultraviolet/themes/dark.css'`

#### The following changes were made:

New script to update the css tokens when updating the tokens from figma

### CODE REVIEW

**Review by DorianMaliszewski — APPROVED** [review]

**Review by atissedrescaleway — APPROVED** [review]

File: packages/ui/src/__stories__/migrations/MigrationSelectInputV2.mdx
Comment (atissedrescaleway) [inline packages/ui/src/__stories__/migrations/MigrationSelectInputV2.mdx:1]: I don't know if this is supposed to be related to this PR ?

**Review by iManu — APPROVED** [review]
Very satisfactory for a first draft, we'll probably come back to submit PRs to optimize the transformation script for our uses when we start import testing. Thank you very much.

File: scripts/create-css-variables.mjs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const screens = {
```
Comment (philibea) [inline scripts/create-css-variables.mjs:1]: This information doesn't come from uvTheme.screen ?

**Comment by lisalupi** [comment]
> hello there 👋
> 
> We don't use these variables for now, but just so you know these values are not correctly generated
> 
> ```
>     --color-other-data: [object Object];
>     --color-other-gradients: [object Object];
>     --color-other-icon: [object Object];
> ```

Fixed ✔️

**Review by philibea — APPROVED** [review]

**Pull request merged**